### PR TITLE
Fix search - trailing comma and trimming spaces

### PR DESCRIPTION
**Pull request by lantisnt**

### CODE REVIEW

File: Modules/LootManager/GUI.lua [hunk 1]
```diff
@@ -142,12 +143,15 @@ local function CreateLootDisplay(self)
     self.st:SetFilter((function(stobject, row)
         local item = strlower(ST_GetItemLink(row))
         local searchKeyword = self.SearchInput:GetText()
-        if searchKeyword and searchKeyword ~= "" and strlen(searchKeyword) >= 3 then
+        if searchKeyword and strlen(searchKeyword) >= 3 then
```
Comment (Peleccotur) [inline Modules/LootManager/GUI.lua:146]: Why not check for trimmed string length here? You are now running some lines of code that makes no sense if someone just enters some spaces

File: Modules/ProfileManager/GUI.lua [hunk 2]
```diff
@@ -89,7 +89,33 @@ local function GenerateUntrustedOptions(self)
             values = filters,
             width = 0.49,
             order = 1
-        }
+        },
```
Comment (Peleccotur) [inline Modules/ProfileManager/GUI.lua:92]: This part is duplicate from the other PR about filtering. Why is it in both?

File: Modules/RosterManager/GUI.lua [hunk 3]
```diff
@@ -108,13 +109,16 @@ local function GenerateUntrustedOptions(self)
             type = "input",
             set = (function(i, v)
                 self.searchString = v
-                if v and v ~= "" and strlen(v) >= 3 then
+                if v and strlen(v) >= 3 then
```
Comment (Peleccotur) [inline Modules/RosterManager/GUI.lua:112]: Same comment on checking trimmed length

**Review by Peleccotur — CHANGES_REQUESTED** [review]

File: Modules/LootManager/GUI.lua (hunk 1 above)
Comment (lantisnt) [reply Modules/LootManager/GUI.lua:146]: We need to detect if anything is input there and there is no computational difference if we check >0 or >N so i used 3 as this is the minimal chars we do search for in a searched entry

Comment (Peleccotur) [reply Modules/LootManager/GUI.lua:146]: discussed -> Trim is not performant because of RegEx check

**Review by Peleccotur — APPROVED** [review]

**Pull request merged**